stravalib/unithelper.py

```diff
@@ -42,27 +42,3 @@ def c2f(celsius):
 
     """
     return (9.0 / 5.0) * celsius + 32
-
-
-def timedelta_to_seconds(td):
```

Comment:
@jsamoocha this pr is really helping me understand how you created the tests and the mock object. super clever! 

I thijnk it's great to remove this as it's built in but we should update the docs with unit conversions if they don't exist. so i'm going to open an issue about that right now. i know we have a ton of of issues but we can still keep chipping away!